nmrfx-core/src/main/java/org/nmrfx/chemistry/io/PDBFile.java

```diff
@@ -218,6 +218,10 @@ public MoleculeBase read(String fileName)
                         molecule.addEntity(compound, coordSetName);
                     }
 
+                    if (compound == null) {
+                        throw new MoleculeIOException("didn't form compound");
+                    }
+
```

Comment:
Followed same format as Polyer exception earlier in function: https://github.com/nanalysis/nmrfx/blob/4145ec117ba89f6926ff2be1dc000cdcaaa25900/nmrfx-core/src/main/java/org/nmrfx/chemistry/io/PDBFile.java#L175